src/reflect/scala/reflect/internal/Names.scala

```diff
@@ -398,11 +398,28 @@ trait Names extends api.Names {
       false
     }
 
+    def lastIndexOf(s: String): Int = {
```

Comment:
how does this compare to java implemention in `String`/`StringBuilder` - 
```
static int lastIndexOf(char[] source, int sourceOffset, int sourceCount,
            char[] target, int targetOffset, int targetCount,
            int fromIndex) {
```

Reply:
If someone chimes in that this is a good idea, I'll benchmark and optimise. This is a first cut at something reasonable. We're comparing a String (which may be short um `"$$"`) with our array.

Reply:
I guess there are only so many ways to look for a string backwards.